source/integrations/zapier.rst

```diff
@@ -0,0 +1,147 @@
+Zapier and Mattermost
+===================
+
+Integrate Mattermost with over 500 supported apps on Zapier, including `GitHub <https://zapier.com/zapbook/github/>`_, `Jira <https://zapier.com/zapbook/jira/>`_, `Wufoo <https://zapier.com/zapbook/wufoo/>`_, `Salesforce <https://zapier.com/zapbook/salesforce/>`_, `Twitter <https://zapier.com/zapbook/twitter/>`_, `Gmail <https://zapier.com/zapbook/gmail/>`_ and `many more <https://zapier.com/zapbook/>`_. Zapier integration triggers new Mattermost messages for events in other apps. This documentation includes:
```

Comment:
Can we clarify what Zapier is a little more, for those who don't know? 

Maybe add in "Zapier is a service that ..... " before the listed integrations. 

Also it should probably be "The Zapier integration triggers"



"Zapier integration triggers new Mattermost messages for events in other apps."